sdk/cmake/Platform/Windows-MSVC.cmake

```diff
@@ -193,8 +193,13 @@ else()
 endif()
 
 if (CMAKE_COMPILER_SUPPORTS_PDBTYPE)
-  set (CMAKE_EXE_LINKER_FLAGS_DEBUG_INIT "/debug /pdbtype:sept")
-  set (CMAKE_EXE_LINKER_FLAGS_RELWITHDEBINFO_INIT "/debug /pdbtype:sept ${MSVC_INCREMENTAL_YES_FLAG}")
+  if(NOT USE_CLANG_CL)
+    set (CMAKE_EXE_LINKER_FLAGS_DEBUG_INIT "/debug /pdbtype:sept")
+    set (CMAKE_EXE_LINKER_FLAGS_RELWITHDEBINFO_INIT "/debug /pdbtype:sept ${MSVC_INCREMENTAL_YES_FLAG}")
+  else()
+    set(CMAKE_EXE_LINKER_FLAGS_DEBUG_INIT "/debug")
+    set(CMAKE_EXE_LINKER_FLAGS_RELWITHDEBINFO_INIT "/debug")
+  endif()
```

Comment:
They don't support pdbtype yet?
lgtm.

Reply:
The build spits dbg files right now, and clang-cl PDB support is only codeview IIRC.